geojson/geometry.py

```diff
@@ -8,6 +9,12 @@ class Geometry(GeoJSON):
     Represents an abstract base class for a WGS84 geometry.
     """
 
+    if (sys.version_info[0] == 3):
+        # Python 3.x has no long type
+        JSON_compliant_types = (float, int, Decimal)
+    else:
+        JSON_compliant_types = (float, int, Decimal, long)
```

Comment:
Try changing this to:

```python
JSON_compliant_types = (float, int, Decimal, long)  # noqa
```

flake8 is throwing the error about `long` not existing in Python 3. `# noqa` tells flake8 to not report errors on this line. Sorta hacky, but it should work for now